vkcs/blockstorage/resource_volume_test.go

```diff
@@ -301,3 +325,55 @@ resource "vkcs_blockstorage_volume" "volume_1" {
   volume_type = "{{.VolumeType}}"
 }
 `
+
+const testAccBlockStorageVolumeOnlineRetype = `
+{{.BaseImage}}
+
+resource "vkcs_blockstorage_volume" "bootable" {
+  name              = "bootable-tf-acc"
+  size              = 10
+  volume_type       = "ceph-hdd"
+  image_id          = data.vkcs_images_image.base.id
+  availability_zone = "GZ1"
+}
+
+resource "vkcs_compute_instance" "basic" {
+  name 				= "instance_tf_acc"
+  flavor_name       = "{{.FlavorName}}"
+
+  block_device {
+    boot_index       = 0
+    source_type      = "volume"
+    destination_type = "volume"
+    uuid             = vkcs_blockstorage_volume.bootable.id
+  }
+
+   network_mode  = "none"
+}
+`
+
+const testAccBlockStorageVolumeOnlineRetypeUpdate = `
+{{.BaseImage}}
+
+resource "vkcs_blockstorage_volume" "bootable" {
+  name              = "bootable-tf-acc"
+  size              = 10
+  volume_type       = "ceph-ssd"
+  image_id          = data.vkcs_images_image.base.id
+  availability_zone = "GZ1"
+}
+
+resource "vkcs_compute_instance" "basic" {
+  name 				= "instance_tf_acc"
+  flavor_name       = "{{.FlavorName}}"
+
+  block_device {
+    boot_index       = 0
+    source_type      = "volume"
+    destination_type = "volume"
+    uuid             = vkcs_blockstorage_volume.bootable.id
+  }
+
+   network_mode  = "none"
+}
+`
```

Comment:
Let's just render it with a {{.VolumeType}} argument. I think that's a bit too much to clone the whole configuration for the single parameter.

Reply:
done